tests/Tests/800__ocm/800__ocm.robot

```diff
@@ -0,0 +1,35 @@
+*** Settings ***
```

Comment:
We discussed in the ODS QE Automation chat room and agreed that, as test ODS-1303 has Component = deploy & Subcomponent = installation in Polarion, this test should be placed in the tests/Tests/100__deploy/100__installation/ folder, probably in tests/Tests/100__deploy/100__installation/102__post_install.robot

Reply:
the documentation link which is checked here is intended to be consumed by the users before installing, so I would keep the same logic and not move this to `102__post_install`. 
Maybe it can just go inside `101_installation`

Reply:
101_installation is fine for me